cosmos/azure-spring-data-cosmos/spring2-samples/cosmos-multi-database-multi-account/src/main/resources/application.yaml

```diff
@@ -19,4 +19,4 @@ spring:
     url: jdbc:mysql://${AZURE_MYSQL_HOST}.mysql.database.azure.com:3306/db_example
   jpa:
     hibernate:
-      ddl-auto: update
+      ddl-auto: update
```

Comment:
add a new line

Reply:
Where do you want a new line? This file wasn't changed, it is the same as before so not sure why it is showing as changed.

Reply:
Done.